Source/WebCore/page/scrolling/mac/ScrollerPairMac.h

```diff
@@ -72,6 +72,7 @@ class ScrollerPairMac : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<S
 
     FloatSize visibleSize() const;
     bool useDarkAppearance() const;
+    ScrollbarWidth scrollbarWidthStyle() const;
```

Comment:
Is this actually called anywhere?

Reply:
My bad it's meant to be called inside of ScrollerMac. I'll fix that now

Reply:
It is now